src/apps/Plotter.jsx

```diff
@@ -0,0 +1,114 @@
+import functionPlot from "function-plot";
+import { useState } from "react";
+import styled from "styled-components";
+
+//------------Styled Components-------------
+
+const Form = styled.form`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+`;
+
+const Input = styled.input`
+  margin: 2px;
+`;
+
+const H3 = styled.h3`
+  color: red;
+`;
+
+//=========================================
+
+let contentsBounds = document.body.getBoundingClientRect();
+let width = 1500;
+let height = 500;
+let ratio = contentsBounds.width / width;
+width *= ratio;
+height *= ratio;
+
+function Renderer() {
+  const [text, changeText] = useState("x^2");
+  const [x_axis, changex] = useState(10);
+  const [y_axis, changey] = useState(5);
+  const [error, changeError] = useState("");
+
+  function handleSubmit(e) {
+    try {
+      e.preventDefault();
+      functionPlot({
+        target: "#instant_apps_root",
+        width,
+        height,
+        xAxis: { domain: [-x_axis, x_axis] },
+        yAxis: { domain: [-y_axis, y_axis] },
+        grid: true,
+        data: [
+          {
+            fn: text,
+          },
+        ],
+      });
+      changeError("");
+    } catch {
+      changeError("Wrong Input, Please Correct it...");
+    }
+  }
+
+  return (
+    <Form onSubmit={handleSubmit}>
+      <label>
+        Enter function
+        <Input
+          type="text"
+          value={text}
+          onChange={(e) => changeText(e.target.value)}
+        ></Input>
+      </label>
+      <label>
+        X-axis Limits
+        <Input
+          type="number"
+          value={x_axis}
+          onChange={(e) => changex(e.target.value)}
+        ></Input>
+      </label>
+      <label>
+        Y-axis Limits
+        <Input
+          type="number"
+          value={y_axis}
+          onChange={(e) => changey(e.target.value)}
+        ></Input>
+      </label>
+      <Input type="submit" value="Plot It!"></Input>
+      <H3>{error}</H3>
+    </Form>
+  );
+}
+
+const shouldRunMyApp = async ({ query }) => {
+  const triggerQueries = ["plot", "plot graph", "Plotter", "graph"];
+
+  for (const triggerQuery of triggerQueries) {
+    if (query.toLowerCase() === triggerQuery) {
+      return { query };
+    }
+  }
+
+  return;
+};
```

Comment:
Words like "plot", "graph" seem to be generic for this app, what do you think? I am thinking of keeping it limited to"graph generator", "plot graph", "plotter".

We also have utility functions for query matching to make things easier, one of them used here: https://github.com/felvin-search/instant-apps/blob/master/src/apps/WordCounter.tsx#L34-L40

Reply:
Yea Sure , those names can conflict with the users actual needs...Making the required changes,

Reply:
@rajivharlalka Can you implement these changes and then we can merge.

Reply:
Oh sorry, making these changes right away.....